django_project/ford3/views/views.py

```diff
@@ -1,3 +1,9 @@
-from django.shortcuts import render # noqa
+# coding=utf-8
 
-# Create your views here.
+from django.conf import settings
+from formtools.wizard.views import CookieWizardView
+
+
+class CampusWizard(CookieWizardView):
```

Comment:
It would be better if it was made into its own file.

Reply:
I agree. But then again the file will only contain these (there's no other view for Campus).
So, I was thinking of changing the `views` python file to something more generic, but doesn't have one in mind for now.